clueless/views.py

```diff
@@ -179,21 +179,35 @@ def playgame(request, game_id):
 	return HttpResponse(template.render(context,request))
 
 @login_required
+
 def playerturn(request, game_id):
+	context = {}
 	template = loader.get_template('clueless/playerturn.html')
 
-	game = Game.objects.get(id=game_id)
-	context = {}
+	#get request variables
+	user_id = request.user
+	#mocked for now, will get for real in jerrold's branch
+	game = Game.objects.get(id = game_id)
 	context['game'] = game
+	player = Player.objects.get(user = user_id, currentGame=game)
+
+	if request.method == 'GET':
```

Comment:
Need to remove the isPlayerTurn from inside the "GET" if statement.  When it's inside this GET if statement, the isPlayerTurn context entry is not set on POST requests, which results in a "It is not your turn" error, even it is my turn.

```
if player.compare(game.currentTurn.player):
    context['isPlayerTurn'] = True
else:
    context['isPlayerTurn'] = False

if request.method == 'GET': 		
    context['roomObjects'] = Room.objects.all()
```